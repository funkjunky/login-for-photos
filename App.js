import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

import Login from './Login';

export default () => (
  <View style={ styles.container }>
    <Text style={ styles.header }>Login For Photos</Text>
    <Login />
  </View>
);

const styles = StyleSheet.create({
  container: {
    display: 'flex',
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'space-around',
    padding: 50,
  },
  header: {
    fontSize: 60,
    textAlign: 'center',
    flexGrow: 1,
  },
});
