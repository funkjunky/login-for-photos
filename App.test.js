import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

describe('App component testing', () => {
  it('renders two text inputs (username and password)', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
