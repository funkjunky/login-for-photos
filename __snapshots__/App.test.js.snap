// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component testing renders two text inputs (username and password) 1`] = `
<View
  style={
    Object {
      "flexGrow": 2,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 50,
        "textAlign": "center",
      }
    }
  >
    Login
  </Text>
  <TextInput
    allowFontScaling={true}
    placeholder="Username"
    style={
      Object {
        "fontSize": 40,
        "minWidth": 185,
      }
    }
    textContentType="username"
  />
  <TextInput
    allowFontScaling={true}
    placeholder="Password"
    secureTextEntry={true}
    style={
      Object {
        "fontSize": 40,
        "minWidth": 185,
      }
    }
    textContentType="password"
  />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffbcbc",
          "borderColor": "red",
          "borderWidth": 1,
          "height": 100,
          "justifyContent": "center",
          "margin": 10,
          "width": 300,
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#333333",
          "margin": 20,
        }
      }
    >
      TouchableNativeFeedback is not supported on this platform!
    </Text>
  </View>
</View>
`;
